jobs/pipelines/powervs/ipi/4.13/daily-ipi4.13-powervs-sydney05/Jenkinsfile

```diff
@@ -8,15 +8,24 @@ pipeline {
     }
     environment {
         //users and credentials. All must be defined in Jenkins Credentials
-        IBMCLOUD_API_KEY = credentials('IBMCLOUD_UPSTREAM_CI_API_KEY_1')
+        PULL_SECRET = credentials('PULL_SECRET')
         GITHUB_USER = credentials('GITHUB_USER')
         GITHUB_TOKEN = credentials('GITHUB_TOKEN')
-        PULL_SECRET = credentials('PULL_SECRET')
         IBMID="ltccci@in.ibm.com"
-        IBMCLOUD_OCCMICCC_API_KEY=credentials('IBMCLOUD_UPSTREAM_CI_API_KEY_1')
-        IBMCLOUD_OIOCCC_API_KEY=credentials('IBMCLOUD_UPSTREAM_CI_API_KEY_1')
-        IBMCLOUD_OMAPCC_API_KEY=credentials('IBMCLOUD_UPSTREAM_CI_API_KEY_1')
-        IBMCLOUD_OCCDIPCCC_API_KEY=credentials('IBMCLOUD_UPSTREAM_CI_API_KEY_1')
+        IBM_APIKEY=credentials('IBMCLOUD_UPSTREAM_CI_API_KEY_1')
+        IBMCLOUD_API_KEY="${IBM_APIKEY}"
+        IC_API_KEY="${IBM_APIKEY}"
```

Comment:
get above two directly from `credentials('IBMCLOUD_UPSTREAM_CI_API_KEY_1')`

Reply:
Changed it